contrib/ros/robots/robot.py

```diff
@@ -0,0 +1,33 @@
+"""
+robot Interface
+"""
+
+
+class Robot(object):
+    def move_to_start_position(self):
```

Comment:
Do all robots have a start position? Is this better called `reset()`? What if I want different start positions for different environments?

I'm not sure this is part of a general robot interface unless it is `reset()`